tests/mocha/blocks/procedures_test.js

```diff
@@ -25,7 +25,7 @@ import {defineRowBlock} from '../test_helpers/block_definitions.js';
 suite('Procedures', function () {
   setup(function () {
     sharedTestSetup.call(this, {fireEventsNow: false});
-    this.workspace = Blockly.inject('blocklyDiv', {});
+    this.workspace = Blockly.inject('blocklyDiv', {disable: true});
```

Comment:
Which test did this change target? Since it's in setup for a suite, I think it will impact multiple tests.

Reply:
I've updated the test to only modify options in the one affected test method.